Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -362,6 +363,69 @@ NonnullRefPtr<StyleValue> property_initial_value(PropertyID property_id)
     return *initial_values[to_underlying(property_id)];
 }
 
+NonnullRefPtr<StyleValue> property_computed_value(PropertyID property_id, Array<RefPtr<StyleValue>, to_underlying(last_property_id) + 1> const& resolved_values)
+{
+    switch (property_id) {
+    // For longhands, simply return the resolved value.
```

Comment:
Do you really mean `resolved` here? As in https://www.w3.org/TR/cssom-1/#resolved-values